sky/sdk/example/widgets/card_collection.dart

```diff
@@ -57,21 +64,50 @@ class CardCollectionApp extends App {
   Widget builder(int index) {
     if (index >= cardModels.length)
       return null;
-    CardModel card = cardModels[index];
 
-    return new Dismissable(
-      key: card.key,
+    CardModel cardModel = cardModels[index];
+    Widget card = new Dismissable(
       onResized: () { layoutState.invalidate([index]); },
-      onDismissed: () { dismissCard(card); },
+      onDismissed: () { dismissCard(cardModel); },
       child: new Card(
-        color: card.color,
+        color: cardModel.color,
         child: new Container(
-          height: card.height,
+          height: cardModel.height,
           padding: const EdgeDims.all(8.0),
-          child: new Center(child: new Text(card.label, style: cardLabelStyle))
+          child: new Center(child: new Text(cardModel.label, style: cardLabelStyle))
         )
       )
     );
+
+    Widget backgroundText = new Center(
+        child: new Text("Swipe in either direction", style: backgroundTextStyle)
+    );
+
+    // The background Widget appears behind the Dismissable card, when the card
+    // moves to the left or right. The Positioned widget ensures that the
+    // size of the background,card Stack will be based only on the card. The
+    // Viewport ensures that when the card's resize animation occurs, the
+    // background (text and icons) will just be clipped, not resized.
+    Widget background = new Positioned(
+      top: 0.0,
+      left: 0.0,
```

Comment:
I take that back. I've left  right:0.0, bottom:0.0 out for now, since if I specify them the background widget disappears (I guess because its size is 0x0). There may be a bug.